src/controllers/update/owned.land.filter.controller.ts

```diff
@@ -0,0 +1,41 @@
+import { NextFunction, Request, Response } from "express";
+
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { ApplicationData } from "../../model";
+import { getApplicationData } from "../../utils/application.data";
+import { OwnedLandKey } from "../../model/owned.land.filter.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `GET ${config.RELEVANT_PERIOD_OWNED_LAND_FILTER_PAGE}`);
+    const appData: ApplicationData = getApplicationData(req.session);
+
+    return res.render(config.RELEVANT_PERIOD_OWNED_LAND_FILTER_PAGE, {
+      backLinkUrl: config.UPDATE_OVERSEAS_ENTITY_CONFIRM_URL,
+      templateName: config.RELEVANT_PERIOD_OWNED_LAND_FILTER_PAGE,
+      ...appData,
+      dateOfCreation: appData.update?.date_of_creation
+    });
+  } catch (error) {
+    logger.errorRequest(req, error);
+    next(error);
+  }
+};
+
+export const post = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `POST ${config.RELEVANT_PERIOD_OWNED_LAND_FILTER_PAGE}`);
+    const ownedLandKey = req.body[OwnedLandKey];
+    // TODO: store ownedLandKey in ApplicationData and turn on questions about the relevant period.
+
+    if (ownedLandKey === '1') {
+      return res.redirect(config.RELEVANT_PERIOD_OWNED_LAND_FILTER_URL); // redirect to interupt screen
+    } else if (ownedLandKey === '0') {
```

Comment:
irrelevant if else. the value is either 1 or 0 no need to check if it is 0. Also es6 allow you to do `!!ownedLandKey` instead of `ownedLandKey === '1'`  but check if git check allows you to use if not you can use `Boolean(ownedLandKey)` in the if statement.

Reply:
good point, thanks